src/routes/landing/components/CityEntry.js

```diff
@@ -62,13 +64,13 @@ class CityEntry extends React.PureComponent<PropType> {
                     underlayColor={theme.colors.backgroundAccentColor}>
         <View>
           <Label theme={theme} searchWords={[filterText]} textToHighlight={city.name} sanitize={normalizeSearchString}
-                 highlightStyle={{ backgroundColor: theme.colors.themeColor }} />
+                 highlightStyle={{ backgroundColor: theme.colors.backgroundColor, fontWeight: 'bold' }} />
```

Comment:
I think `backgroundColor: theme.colors.backgroundColor` is not needed at all, is it? Same for other PRs and the occurrence below.

Reply:
You're right. In the webapp it was neccesary, that's why I assumed it works the same way in the native app, but I was wrong

Reply:
Do you know why it is necessary in the webapp and not in the native app?

Reply:
In the WebApp we use a different highlighter, which has a default color, I think